packages/core/src/components/table/table-component-expandable-rows.stories.tsx

```diff
@@ -130,34 +130,32 @@ export default {
       control: {
         type: 'radio',
       },
-
       options: ['auto', 'hidden'],
```

Comment:
The default value to Overflow seems to have gotten lost along the way, could we maybe set that to "auto"?
<img width="1146" alt="Screenshot 2024-10-24 at 13 08 34" src="https://github.com/user-attachments/assets/51654f58-8377-4dfc-8b9a-9c81e4dc845e">

Reply:
This, but also there should be three values: 'auto' | 'hidden' | 'visible' :)